tests/Behat/AuthenticationContext.php

```diff
@@ -147,4 +147,30 @@ public function allowExternalUsers(): void {
     $this->configContext->setConfig('oe_authentication.settings', 'assurance_level', 'LOW');
   }
 
+  /**
+   * Configures the CAS server so users are created in login if not exists.
+   *
+   * @Given the site is configured to register users if not exists
+   */
+  public function theSiteIsConfiguredToRegisterUsersIfNotExists(): void {
+    $this->configContext->setConfig('user_accounts.auto_register', 'cas.settings', TRUE);
+  }
+
+  /**
+   * Create a local user.
+   *
+   * @Given a user with the same email already exists locally
```

Comment:
We don't need this. Just use "Given users:", create a local use with this email. Then use "Given CAS users" (see my comment in the .feature file) and add a `Local username` column to link the local user. For details, see `CasMockServerContext::registerUsers()`.